src/components/ContactCard/ContactForm/formValuesToContact.js

```diff
@@ -77,16 +71,8 @@ const formValuesToContact = (data, oldContact) => {
     }
   }
 
-  return {
-    ...contact,
-    fullname: getFullname(contact),
-    displayName: getDisplayName(contact),
-    indexes: {
-      byFamilyNameGivenNameEmailCozyUrl: getIndexByFamilyNameGivenNameEmailCozyUrl(
-        contact
-      )
-    }
-  }
+  const normalizedContact = normalizeContact(contactWithFormValues)
+  return normalizedContact
```

Comment:
Nit: We can remove this variable assignation and `return normalizeContact(contactWithFormValues)` directly.